### ISSUE TITLE
Log file not being generated

### ISSUE THREAD

**Issue by kmancusi** [applitools/ImageTester#15]
Using latest version of ImageTester, when I execute command:
`java -jar ImageTester_3.2.2.jar -k{APPLITOOLS_API_KEY -fbaselines -lf 'applitools-logging.log' -a 'PJN-PDFs' -fb 'pjn-redundancy-testing-poc' -te`
Where `-lf` is supposed to be for a log file

I am not getting a log file generated.

**Comment by lev-at** [comment]
@kmancusi You are right, under the hood there was a change in the Eyes SDK, now all logs are automatically generated in 

Mac/Linux
`$TMPDIR/applitools-logs/`

Windows
`$env:TEMP/applitools-logs/`

The log directory can now be specified using this environment variable:
`APPLITOOLS_LOG_DIR=<path>`

Thanks for letting us know, I will make sure the flag and docs will reflect the new changes.
